go.mod

```diff
@@ -168,7 +168,7 @@ require (
 	github.com/antihax/optional v1.0.0
 	github.com/containers/image/v5 v5.25.0
 	github.com/rancher/rancher/pkg/apis v0.0.0-20230915232223-a9ea4ce4a5ba
-	github.com/rancher/shepherd v0.0.0-20240130172509-3690b7e7016b
+	github.com/rancher/shepherd v0.0.0-20240206201006-f299197dc871
```

Comment:
Hey you don't need to do a replace? Since the GH is complaining about the go mod, I'm curious if this actually doing it properly

Reply:
Yeah I talked to Caleb about this. I think as a best practice to avoid accidently putting in our forked repo to rancher/rancher, after I tested the freeform jobs, I just changed the `go.mod` to what should be expected once the PR is merged.

I had the replace before when testing the jobs, then after I made sure all was good, I just changed it to what we see now.

Reply:
Or are we expected to just have the replace in the go.mod in the rancher/rancher PRs and then just change them after the shepherd PRs go in. Unclear on the process here...

Reply:
this PR is pending on shepherd changes, right? So you can't know the commit (can you?) before that merges? 
IMO for safety we should leave the commit as it is on upstream rancher for a given branch. in other words, don't change it until the PR in shepherd has merged and we have a new version for it